index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Marshall's Quiz 1</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<header class="mainHead">
+		<h1>My Elephant Products</h1>
+	</header>
+<div class="contentContainer">
+	<article>
+		<!-- Product 1 -->
+		<header >
+			<h1 class="prodTitle">Elephant Razor</h1> <!-- Product Name -->
+		</header>
+		<!-- Image section -->
+		<section class="sectionOne">
+			<img src="https://static.boker.de/us/images/zoom/140510.jpg" class="prodImg" alt="The Razor" width="80%" height="100px"> <!-- Product image -->
```

Comment:
Use CSS to change the display properties of an `img` element instead of using the HTML attributes.